src/outlet_registry.js

```diff
@@ -59,4 +59,9 @@ OutletRegistry.prototype.update = function(outlet_id, component) {
     }
 };
 
+OutletRegistry.prototype.rewind = function(){
+    this.outlets = [];
+    this._unique_id_gen = 0;
```

Comment:
you don't need to reset this - it just needs to provide unique ids, and will eventually wrap around when it hits int max